app/controllers/shipit/tasks_controller.rb

```diff
@@ -22,8 +22,14 @@ def show
     end
 
     def create
-      @task = stack.trigger_task(params[:definition_id], current_user)
-      redirect_to [stack, @task]
+      @definition = stack.find_task_definition(params[:definition_id])
+
+      if @definition.allow_concurrency? || params[:force] || !@stack.deploying?
```

Comment:
Yeah I wondered about it. My though was that `deploying?` while not 100% accurate is relatively good. However I agree `active_deploy` should be renamed in `active_task`. Will do.